gravitee-node-management/src/main/java/io/gravitee/node/management/http/node/log/LogLevelEndpoint.java

```diff
@@ -0,0 +1,66 @@
+package io.gravitee.node.management.http.node.log;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import io.gravitee.common.http.HttpMethod;
+import io.gravitee.common.http.HttpStatusCode;
+import io.gravitee.common.http.MediaType;
+import io.gravitee.node.management.http.endpoint.ManagementEndpoint;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.ext.web.RoutingContext;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Remi Baptiste (remi.baptiste at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Slf4j
+public class LogLevelEndpoint implements ManagementEndpoint {
+
+    @Override
+    public HttpMethod method() {
+        return HttpMethod.GET;
+    }
+
+    @Override
+    public String path() {
+        return "/loglevel";
+    }
+
+    @Override
+    public void handle(RoutingContext context) {
+        final String loggerName = context.request().getParam("logger");
```

Comment:
I'm wondering if we could extend it to a `/logging` endpoint so we could have a GET to display the current config and a POST to set up log levels for several loggers at once?
In both cases (GET and POST), it could be a simple `Map<String,String>` where the key is the logger name and the value is the log level. WDYT?